hummingbot/connector/exchange/binance/binance_constants.py

```diff
@@ -70,9 +70,9 @@
 RATE_LIMITS = [
     # Pools
     RateLimit(limit_id=REQUEST_WEIGHT, limit=1200, time_interval=ONE_MINUTE),
-    RateLimit(limit_id=ORDERS, limit=50, time_interval=10 * ONE_SECOND),
-    RateLimit(limit_id=ORDERS_24HR, limit=160000, time_interval=ONE_DAY),
-    RateLimit(limit_id=RAW_REQUESTS, limit=6100, time_interval= 5 * ONE_MINUTE),
+    RateLimit(limit_id=ORDERS, limit=10, time_interval=10 * ONE_SECOND),
+    RateLimit(limit_id=ORDERS_24HR, limit=100000, time_interval=ONE_DAY),
+    RateLimit(limit_id=RAW_REQUESTS, limit=6000, time_interval= 5 * ONE_MINUTE),
```

Comment:
I do remember us talking about the limits being fetched using from an endpoint, but I still feel that it would be right, to use the correct values here.
```suggestion
    RateLimit(limit_id=ORDERS, limit=50, time_interval=10 * ONE_SECOND),
    RateLimit(limit_id=ORDERS_24HR, limit=160000, time_interval=ONE_DAY),
    RateLimit(limit_id=RAW_REQUESTS, limit=6100, time_interval= 5 * ONE_MINUTE),
```

Reply:
You are right with this limits